statuses.py

```diff
@@ -73,5 +73,9 @@
    "I love hugging. I wish I was an octopus so I could hug ten people at a time.",
    "Hug me and you will see the moon. Love me and I will bring the moon to you!",
    "Hugs and kisses are ways to express what cannot be said.",
-   "Sometimes all you can do is hug a friend tightly and wish that their pain could be transferred by touch to your own emotional hard drive."
+   "Sometimes all you can do is hug a friend tightly and wish that their pain could be transferred by touch to your own emotional hard drive.",
+   "In your darkest, hug someone. It helps!",
+   "Live, Love, Hug",
+   "Live is simply better when you receive and give a hug",
+   "Go out and hug someone today!"
```

Comment:
The convention I was following was having a trailing comma after the last sentence in the list (it makes for cleaner diffs in subsequent commits/PRs).

(I realize the last person forgot to add it, and i guess it's on me for not catching it, but lets add it here for the future).

Reply:
Thanks for catching these errors early. I have made the changes, please check them out.

Reply:
Thanks! Looks good! 🤗